content/en/user-manual/graphics/lighting/lightmapping.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Lightmapping
+template: usermanual-page.tmpl.html
+position: 3
+---
+
+TODO: make image clickable
+![PlayCanvas Lightmapping][1]
+*The lighting in this scene is provided by Lightmap and AO textures and IBL (reflections)*
+
+Here is the link to final scene that uses these techniques to achieve results in image above: External HDR Lightmaps and Ambient Occlusiong rendered using 3D's Max Vray and HDR CubeMap applied using Box Projection to enable IBL ([Image Based Lighting][11]) reflections.
+
+TODO: link to AO doc
+TODO: link to project
+
+## Overview
+
+Lightmap generation is the process of pre-calculating lighting information for a static scene and storing it in textures which are then applied on materials. This is an efficient way of lighting a scene if much of the light sources and geometry are static or environmental. And produces realistic results.
+
+PlayCanvas offers two ways to use lightmaps in your scene: **External lightmap generation** - which is discussed in this page below. And [**Runtime Lightmapping**][0] that can be generated automatically by the Engine on loading or later while application is running.
+
+## External Lightmap Generation
+
+Many 3D content tools have ways to generate lightmap textures. For example, 3DS Max, Maya, Blender and other tools all have ways to bake lightmaps into textures. The advantages of using an offline tool for lightmap generation is that you can use very sophisticated lighting calculations like Global Illumination for bounce lighting, soft shadows, ambient occlusion, etc. The major disadvantage is that you have to have a complete representation of your scene inside the 3D tool. So if your PlayCanvas scene is made up of lots of instances positioned in the Editor, you need to re-create this inside your lightmapping tool.
+
+Once you have created lightmaps using an external tool you simply upload them as regular texture assets and they can be added to your materials using the lightmap slot in the standard Physical Material.
+
+**Read further to get more details and best practices on rendering lightmaps.**
+
+## Tools
+
+Various 3D tools can be used to render lightmaps. In this page we will use 3D Studio Max as example, but same functionality is achievable by any other similar modeling tools.
+
+## Gamma Correction
+
+When rendering Lightmaps or CubeMaps they should be rendered in Linear Space, to ensure colour curves are not affected by gamma correction twice. PlayCanvas Engine will apply gamma correction during real-time rendering.
```

Comment:
Is there any reason to use Captial Letters for many words? Linear space, global illumination, etc